conf/modules/teraranger_one.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="teraranger_one" dir="range_finder">
+  <doc>
+    <description>Driver for the TeraRanger One range finder (I2C)</description>
+    <configure name="TERARANGER_ONE_I2C_DEV" value="i2cX" description="Select I2C bus"/>
+    <define name="TERARANGER_ONE_I2C_ADDR" value="0x30" description="Select an alternate I2C address"/>
+    <define name="TERARANGER_ONE_OFFSET" value="0.0" description="Add an offset to the measurments" unit="m"/>
+    <define name="USE_TERARANGER_ONE_AGL" value="TRUE|FALSE" description="Send data as AGL measurments to the system"/>
```

Comment:
This is for the case where you don't want to use the sensor to detect ground but something else, like obstacle in front of you. In this case you don't want to send the ABI AGL message.